posts/formalizing-isocrystals-in-lean.md

```diff
@@ -0,0 +1,218 @@
+---
+author: 'Robert Y. Lewis, Heather Macbeth'
+category: 'New in mathlib'
+date: 2022-07-19 19:30:08 UTC+02:00
+description: ''
+has_math: true
+link: ''
+slug: classification-of-one-dimensional-isocrystals
+tags: ''
+title: Classification of one-dimensional isocrystals
+type: text
+---
+
+Last year, there was a [big mathlib refactor](link://filename/posts/semilinear-maps.md) to replace linear maps throughout the library with  *semilinear maps*,
```

Comment:
In https://github.com/leanprover-community/blog/blob/isocrystals/posts/formalizing-isocrystals-in-lean.md the hyperlink is not showed.

Reply:
This syntax is specific to Nikola; it works when the blog is built!